### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by PabloCastellano**
Result: https://github.com/Blobscan/blobscan-indexer.rs/blob/up-docs/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,61 +4,80 @@ The indexer for the [Blobscan](https://github.com/Blobscan/blobscan) explorer im
...
 
 ```bash
 docker run --rm blossomlabs/blobscan-indexer:master
 ```
 
-## Environment variables
+Using environment variables
+
+```bash
+docker run -e BLOBSCAN_API_ENDPOINT=http://blobscan-api:3001 -e BEACON_NODE_RPC=http://beacon:3500 -e EXECUTION_NODE_URL=http://execution:8545 --rm blossomlabs/blobscan-indexer:master
```
Comment (0xGabi) [inline README.md:60]: I think we are repeating the `--rm blossomlabs/blobscan-indexer:master` section twice.

**Review by 0xGabi — CHANGES_REQUESTED** [review]
Looks good. I left a single comment to address before we can merge.

File: README.md (hunk 1 above)
Comment (PabloCastellano) [reply README.md:60]: why? these two commands are equivalent ways of running the container with environment variables

Comment (PabloCastellano) [reply README.md:60]: I've added a comment for the second command. Let me know if it's clearer now!

**Pull request merged**